test/sa_2d_turbulent_channel_regression.sh.in

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+PROG="@top_builddir@/test/test_turbulent_channel"
+
+INPUT="@top_srcdir@/test/input_files/sa_2d_turbulent_channel_regression.in"
+
+MESH_1D='@top_srcdir@/test/test_data/turbulent_channel_Re944_grid.xda'
+DATA_1D='@top_srcdir@/test/test_data/turbulent_channel_soln.xda'
+
+DATA='@top_srcdir@/test/test_data/sa_2d_turbulent_channel_regression.xdr'
+
+PETSC_OPTIONS="-pc_type asm -pc_asm_overlap 2 -sub_pc_type lu -sub_pc_factor_mat_solver_package mumps"
```

Comment:
Can we experiment with the same tons-of-overlap, tons-of-ilu conbinations that worked for the other troublesome cases?  I went to a lot of trouble to avoid adding a MUMPS dependency to "make check"...